easybuild/framework/easyblock.py

```diff
@@ -3466,8 +3472,15 @@ def make_checksum_lines(checksums, indent_level):
 
                     for key, val in sorted(ext_options.items()):
                         if key != 'checksums' and val != exts_default_options.get(key):
-                            val = quote_str(val, prefer_single_quotes=True)
-                            exts_list_lines.append("%s'%s': %s," % (INDENT_4SPACES * 2, key, val))
+                            strval = quote_str(val, prefer_single_quotes=True)
+                            line = "%s'%s': %s," % (INDENT_4SPACES * 2, key, strval)
+                            # fix long lines for list-type values (e.g. patches)
+                            if len(line) > MAX_LINE_LENGTH and isinstance(val, list):
```

Comment:
What about this?

`if len(val) > 1 and isinstance(val, list):`

Reply:
makes sense, fixed in b2ee70abf